test/taxjar.spec.js

```diff
@@ -81,6 +82,16 @@ describe('TaxJar API', () => {
       assert.equal(taxjarClient.getApiConfig('apiUrl'), 'https://api.sandbox.taxjar.com/v2/');
     });
 
+    it('includes appropriate headers', () => {
+      nock(taxjarClient.getApiConfig('apiUrl'), {allowUnmocked: true}).get('/rates/12345').reply(function() {
```

Comment:
We should use the `() => { ... }` syntax over `function() { ... }` where-ever possible.

Reply:
@codymullins `function` is necessary/intentional in this case since we need to access `this` in the right context for `this.req.headers` rather than the lexical scope. See the "No separate this" section [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) for more.

Reply:
I probably should have clicked the link you sent; I thought `req` was ours. 🙂